packages/um/package.py

```diff
@@ -150,6 +153,14 @@ class Um(Package):
             "fcm_name": "netcdf",
             "fcm_ld_flags": "-lnetcdff -lnetcdf"}}
 
+    # Optional JULES sources to be used in build (i.e. AM3)
+    jules_git_url = "https://github.com/ACCESS-NRI/JULES.git"
+    variant("jules_sources", default="NA", values=str, multi=False, description=f"Optional JULES sources branch/tag/commit from {jules_git_url}.")
```

Comment:
The string that the Spack documentation uses is "none": https://spack.readthedocs.io/en/latest/packaging_guide_creation.html#multi-valued-variants